src/tcpip/tcpip.c

```diff
@@ -176,10 +176,7 @@ static void onstatechange(struct mg_tcpip_if *ifp) {
   if (ifp->state == MG_TCPIP_STATE_READY) {
     MG_INFO(("READY, IP: %M", mg_print_ip4, &ifp->ip));
     MG_INFO(("       GW: %M", mg_print_ip4, &ifp->gw));
-    if (ifp->lease_expire > ifp->now) {
-      MG_INFO(
-          ("       Lease: %lld sec", (ifp->lease_expire - ifp->now) / 1000));
-    }
+    MG_INFO(("      MAC: %M", mg_print_mac, &ifp->mac));
```

Comment:
We are printing the MAC only on those examples where the user would not know it (we generate it).
This is not useful info when the user has set the MAC and makes us print it twice when it is.

Reply:
It actually might be useful to see logs, to spot conflicts or outright bad MAC settings